### PR TITLE
FDS Source: Define VVLES parameters

### PR DESCRIPTION
**Pull request by mcgratta**

### CODE REVIEW

File: Source/read.f90 [hunk 1]
```diff
@@ -1739,45 +1739,36 @@ SUBROUTINE READ_MISC
...
+   VN_MAX = 0.5
+   VN_MIN = 0.4
+   FLUX_LIMITER = CHARM_LIMITER
+   IF (TURBULENCE_MODEL/='null') THEN
+      WRITE(MESSAGE,'(A,A,A)')  'ERROR: TURBULENCE_MODEL,',TRIM(TURBULENCE_MODEL),', is not appropriate for DNS.'
+      CALL SHUTDOWN(MESSAGE) ; RETURN
+   ENDIF
 ELSEIF (RESEARCH_MODE .OR. SIMULATION_MODE=='LES') THEN
    SIM_MODE = LES_MODE
+   FLUX_LIMITER = SUPERBEE_LIMITER
```
Comment (rmcdermo) [inline Source/read.f90:1754]: @mcgratta I think LES should use CHARM (FLUX_LIMITER=2).

**Comment by mcgratta** [comment]
Will do. I'm testing again.

**Comment by rmcdermo** [comment]
Of course, I meant FLUX_LIMITER=CHARM_LIMITER (which is actually 4).

Do we want to take this opportunity to make FLUX_LIMITER a character string for input, instead of the cryptic integer designation?

**Comment by drjfloyd** [comment]
I think that would be a good idea.

**Pull request merged**